### PR TITLE
3663: Updating UT for chassis under svc-system

### PR DESCRIPTION
**Pull request by tamanna-k**
updating UT for chassis under svc-system for Network Adapters collection and individual Network Adapter

### CODE REVIEW

File: svc-systems/chassis/getresource_test.go [hunk 1]
```diff
@@ -354,6 +369,41 @@ func TestPluginContact_GetChassisResource(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "successful get data",
```
Comment (jeevan-kamkar) [inline svc-systems/chassis/getresource_test.go]: rename the test case

File: svc-systems/chassis/getresource_test.go [hunk 2]
```diff
@@ -354,6 +369,41 @@ func TestPluginContact_GetChassisResource(t *testing.T) {
...
+				Header:        header,
+				StatusCode:    http.StatusOK,
+				StatusMessage: response.Success,
+				Body:          map[string]interface{}{"@odata.id": "/redfish/v1/Chassis/6d4a0a66-7efa-578e-83cf-44dc68d2874e:1/NetworkAdapters"},
+			},
+			wantErr: false,
+		},
+
+		{
+			name: "successful get data",
```
Comment (jeevan-kamkar) [inline svc-systems/chassis/getresource_test.go]: rename the test case

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
address the requested changes

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**